PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/springcloud/library/SpringCloudUtils.java

```diff
@@ -0,0 +1,163 @@
+ * the Software.
+ * <p/>
+ * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+ * THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.microsoft.intellij.runner.springcloud.library;
+
+import com.microsoft.azure.credentials.AzureCliCredentials;
+import com.microsoft.azure.management.appplatform.v2019_05_01_preview.AppResourceProperties;
+import com.microsoft.azure.management.appplatform.v2019_05_01_preview.DeploymentResourceProperties;
+import com.microsoft.azure.management.appplatform.v2019_05_01_preview.DeploymentSettings;
+import com.microsoft.azure.management.appplatform.v2019_05_01_preview.PersistentDisk;
+import com.microsoft.azure.management.appplatform.v2019_05_01_preview.UserSourceInfo;
+import com.microsoft.azure.management.appplatform.v2019_05_01_preview.UserSourceType;
+import com.microsoft.azure.management.appplatform.v2019_05_01_preview.implementation.AppPlatformManager;
+import com.microsoft.azure.management.appplatform.v2019_05_01_preview.implementation.AppResourceInner;
+import com.microsoft.azure.management.appplatform.v2019_05_01_preview.implementation.DeploymentResourceInner;
+import com.microsoft.azure.management.appplatform.v2019_05_01_preview.implementation.ResourceUploadDefinitionInner;
+import com.microsoft.azure.storage.file.CloudFile;
+import com.microsoft.azuretools.authmanage.CommonSettings;
+import com.microsoft.intellij.runner.springcloud.SpringCloudConstants;
+import com.microsoft.intellij.runner.springcloud.SpringCloudModel;
+import com.microsoft.rest.LogLevel;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.util.HashMap;
+import java.util.Map;
+
+public class SpringCloudUtils {
+
+    private static Map<String, AppPlatformManager> managerMap = new HashMap<>();
+
+    public static AppPlatformManager getAppPlatformManager(String subscriptionId) throws IOException {
+        if (!managerMap.containsKey(subscriptionId)) {
+            final AppPlatformManager manager = AppPlatformManager.configure()
+                    .withUserAgent(CommonSettings.USER_AGENT)
+                    .withLogLevel(LogLevel.BODY_AND_HEADERS)
+                    .authenticate(AzureCliCredentials.create(), subscriptionId);
+            managerMap.put(subscriptionId, manager);
+        }
+        return managerMap.get(subscriptionId);
+    }
+
+    public static AppResourceInner execute(SpringCloudModel springCloudModel) throws IOException {
+        return null;
```

Comment:
where is 'execute' used?

Reply:
Has refactor this method which should be removed